src/instance/mod.rs

```diff
@@ -18,27 +18,39 @@ use data::{Map,Monster};
 
 pub mod management;
 
+// Computation of the duration of one so-called 'tick'. A 'tick' is a
+// unite in the discrete time of lycan. By default, its value is set
+// to about 16ms.
```

Comment:
This value doesn't make any sense, I already told @Vaelden about it, but basically this number should be based on experimentation, *not* on framerate.
The framerate needs to be under 16ms only because if rendering a frame goes above that number the user might start seeing issues. However here there's **no** correlation between the two.

Having a two low number is basically killing the server. It burns CPU for nothing.
Having a tow high number can introduce glitch and deteriorate the player's experience.

Whatever value, the default needs probably to be computed dynamically and until we reached that point, it should be driven by experimentation. A better number would be 100ms in my opinion.